common/hooks/pre-pkg/06-verify-python-deps.sh

```diff
@@ -3,6 +3,18 @@
 # This hook executes the following tasks:
 #	- Verifies python module dependencies from dist-info's METADATA and egg-info's PKG-INFO
 
+python_provides() {
+    local -a _provides
+    for _pkgname in ${pkgname} ${subpackages}; do
+        [ -r "${XBPS_STATEDIR}/${_pkgname}-provides" ] || continue
+        _provides=($(<"${XBPS_STATEDIR}/${_pkgname}-provides"))
+        for x in "${_provides[@]}"; do
+            case "$x" in py3:*) ;; *) continue ;; esac
+            printf '%s %s\n' "${_pkgname}-${version}" "$x"
```

Comment:
This should should probably be `${_pkgname}-${version}_${revision}`, or we just drop it and make it just pkgnames, but not sure maybe in the future the version will become useful.

Reply:
I think `${_pkgname}-${version}_${revision}` is fine